server/testserver.go

```diff
@@ -196,6 +199,38 @@ func (ts *TestServer) StartWithStopper(stopper *stop.Stopper) error {
 	return nil
 }
 
+// ExpectedInitialRangeCount returns the expected number of ranges that should
+// be on the server after initial (asynchronous) splits have been completed,
+// assuming no additional information is added outside of the normal bootstrap
+// process.
+func ExpectedInitialRangeCount() int {
+	return GetBootstrapSchema().DescriptorCount() - sql.NumSystemDescriptors
+}
+
+// WaitForInitialSplits waits for the server to complete its expected initial
+// splits at startup. If the expected range count is not reached within the
+// supplied timeout, the current test will be aborted.
+func (ts *TestServer) WaitForInitialSplits(t util.Tester, timeout time.Duration) {
```

Comment:
When would a caller pass in a different values for `timeout`? It seems like the timeout constant should be here rather than at each call site. 

Why isn't `WaitForInitialSplits` called from `StartTestServer`?